src/primitives/tweedlesig.rs

```diff
@@ -0,0 +1,36 @@
+//! TODO
+//!
+//! - If we're using the other curve, then this won't be an instantiation of RedDSA
```

Comment:
I'm confused. Yes it will. (RedDSA supports cofactor 1 as a special case, and it's not dependent on curve type.) We should definitely generalize the `redjubjub` crate rather than writing completely new code.

Reply:
Oh, is that so? Then yes, we can generalise.